src/software/visualizer/widgets/robot_status_table.h

```diff
@@ -45,7 +46,16 @@ class RobotStatusTable : public QTableWidget
    public slots:
     /**
      * Updates the table with a new robot status. If the status message already exists,
-     * the message age is refreshed to 0. Othewise, a new message is added with an age of
+     * the message age is refreshed to 0. Otherwise, a new message is added with an age of
+     * 0.
+     *
+     * @param sensor_msg The status to add to the table
+     */
+    void updateSensorMsg(const SensorMsg& sensor_msg);
```

Comment:
This function technically doesn't need the entire SensorMsg, only the `TBotsRobotMsg`. Also because this class doesn't "own" any sensor msg information I think a name similar to the existing one like `updateRobotStatus` is more descriptive of what the function is doing.